koji-blame

```diff
@@ -55,7 +56,7 @@ def run_list_tag_history(item):
         raise Error("%s is not a package, build or tag" % item)
     return utils.sbacktick(cmd)[0]
 
-def parse_history(lth_output, since=EPOCH):
+def parse_history(lth_output, since=EPOCH, until=NOW):
```

Comment:
I suggest default `until=None`, and then later only include `until` in the condition if `until is not None`